analysis/cox_model.do

```diff
@@ -0,0 +1,217 @@
+/*----------------------------------------------------------------------------
+Do file name: 			cox_models
+Project: 				Project 12: Post covid CVD events
+Date:					08/09/2022
+Author:					Venexia Walker and Rachel Denholm
+                        Adapted from post-covid-vaccinated cox_model.do
+Description:			Reformating of CSV file and running cox models
+Datasets used:			csv outcome files
+Datasets created:		*_cox_model.txt
+Other output:			logfiles
+-----------------------------------------------------------------------------*/
+
+local cpf "`1'"
+
+* Set file paths
+
+global projectdir `c(pwd)'
+di "$projectdir"
+
+* Set Ado file path
+
+adopath + "$projectdir/analysis/extra_ados"
+
+* Import and describe data
+
+import delim using "./output/`cpf'.csv" 
+
+des
+
+* Filter data
+
+keep patient_id index_date age_at_cohort_start expo_date region_name follow_up_start event_date ethnicity follow_up_end cox_weights cov_cat* cov_num* cov_bin*
+
+* Rename variables
+rename age_at_cohort_start age
+rename expo_date exposure_date
+rename region_name region
+rename event_date outcome_date
+
+* Generate pre vaccination cohort dummy variable
+gen prevac_cohort = regexm("`cpf'", "_pre")
```

Comment:
[Optional] Please could this be 'prevax' or whatever is consistent with the data files and R code.